store/store_test.go

```diff
@@ -335,6 +337,181 @@ func (t *remoteRepoTestSuite) TestDownloadSyncFails(c *C) {
 	c.Assert(osutil.FileExists(tmpfile.Name()), Equals, false)
 }
 
+type downloadBehaviour []struct {
+	URL   string
+	Error bool
```

Comment:
These can all lowercase. Ideally we should use uppercase specifically when we intend things to be exported somehow, either from the package or via marshaling.